deno/lib/jsr.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@colinhacks/zod",
```

Comment:
Would love to get this shipped & keep moving more & more to JSR for everything.

Though I think this should be changed to `@zod/zod` or something else under the `@zod` scope in JSR. 
 Especially since @colinhacks already owns it on JSR, and would mean it follows the pattern similar to Hono with `@hono/hono`.